Dockerfile

```diff
@@ -77,6 +77,14 @@ RUN git clone https://github.com/woboq/woboq_codebrowser /woboq && \
            -DCMAKE_BUILD_TYPE=Release . \
      make)
 
+# https://github.com/PaddlePaddle/Paddle/issues/8195
+# NCCL2.1.4 seems works well on cuda9, but not compatible with cuda8
+# TODO(dzhwinter): this disable the NCCL DSO temporarily, should be removed
+# Download the NCCL1.3 and build it locally
+RUN git clone https://github.com/NVIDIA/nccl /nccl && \
```

Comment:
Just to make sure, https://github.com/PaddlePaddle/Paddle/blob/develop/tools/manylinux1/Dockerfile.x64#L53 has the same result. `Dockerfile` under root folder is use for CI and development, manylinux1 Dockerfile is used for releasing.

Reply:
This PR only for the CI testing.